logger/logger.py

```diff
@@ -1,10 +1,20 @@
 import logging
+from io import StringIO
 from logger.consts import errors
 
 
 logging.addLevelName(50, "FATAL")
 logging.basicConfig(level=logging.DEBUG, format="%(asctime)s [%(levelname)s] %(message)s", datefmt="%y/%m/%d %H:%M:%S")
 
+# Set up StringIO handler for unit test
+root = logging.getLogger()
+root.setLevel(logging.DEBUG)
+log_stream = StringIO()
+handler = logging.StreamHandler(log_stream)
+handler.setLevel(logging.DEBUG)
+handler.setFormatter(logging.Formatter("%(asctime)s [%(levelname)s] %(message)s", datefmt="%y/%m/%d %H:%M:%S"))
+root.addHandler(handler)
+
```

Comment:
Can you add context why these changes are needed?

Reply:
The functions in `logger.py` do not return any values, they simply output to console. In order to capture this output with `pytest`, the output needs to be in either `stdout` or `stderr` streams. This code creates an identical `StringIO()` logging stream and attaches it to the main logger so that `test_logger.py` has a chance to access the contents of the output through `log_stream`. I cannot access the default logging `stdout` stream because it is automatically emptied after outputting to console. I am essentially duplicating the output of the logger and saving it because the original output of the logger cannot be accessed.

There is probably a better way to do this, but this is the only working solution I can come up with from my research. I may be missing an important concept entirely.